doc/source/serve/deploy-many-models/multi-app.md

```diff
@@ -183,47 +176,17 @@ The command `serve run` blocks the terminal, which allows logs from Serve to str
 
 
 ## Adding, Deleting, and Updating Applications
-The config submitted to the cluster describes the target state for Ray Serve. Consequently, Ray Serve will add, remove, or update based on the list of applications in the config and the config options set for each application.
-* To add an application, add a new entry under the `applications` field.
-* To delete an application, remove the corresponding entry under the `applications` field.
-* To update an application, modify the config options in the corresponding entry under the `applications` field.
+You can add or remove entries under the `applications` field to add or remove applications from the cluster. This will not affect other applications on the cluster. To update an application, modify the config options in the corresponding entry under the `applications` field.
 
 :::{note}
-The update behavior for each application when a config is resubmitted is the same as the old single-application behavior. For how an application reacts to different config changes, see [Updating a Serve Application](serve-inplace-updates).
+The update behavior for an application when a config is resubmitted is the same as the old single-application behavior. For how an application reacts to different config changes, see [Updating a Serve Application](serve-inplace-updates).
```

Comment:
Suggest specifying this is for in-place updates so users get a high level idea without having to click on the next page to understand everything